src/ng-select/ng-select.component.ts

```diff
@@ -398,7 +414,7 @@ export class NgSelectComponent implements OnInit, OnDestroy, OnChanges, AfterVie
     }
 
     onItemHover(item: NgOption) {
-        if (item.disabled) {
+        if (item.disabled || this._arrowKeyPressed) {
```

Comment:
This solution will work, but I wonder are there any other ways to reset mouse pointer events when arrow key is pressed?

Reply:
@andreasfranz @anjmao  I will take a deeper look into this issue to see why this behaviour occurs.

Reply:
this solution doesn't fix all cases. If virtual scroll is disabled it still occurs. E.g: `Data sources` page

Reply:
@varnastadeus Yes you are right, this only fixes the case when you go through the item by holding down the key and not by going through them one by one with the arrow keys.